src/clj/rems/css/styles.clj

```diff
@@ -20,63 +20,14 @@
             [mount.core :as mount]
             [rems.config :refer [env]]
             [rems.context :as context]
+            [rems.css.style-utils :refer [get-theme-attribute get-logo-image generate-at-font-faces get-navbar-logo get-logo-name-sm]]
```

Comment:
I like this refactoring. If we can manage to clean up rems.css.styles so that it only contains the actual styles it would make the codebase a bit cleaner. This is a step in that direction :+1:

Reply:
I think this is a bit premature split but overall one util file is ok.

Reply:
LOL